### DISCUSSION TITLE
How to use symbols - Conventions

### DISCUSSION THREAD

**Discussion by b-gauthier** [squidfunk/mkdocs-material#7061] [Q&A] [answered]
Hello, 
I'm wondering how to use the symbols we find on https://squidfunk.github.io/mkdocs-material/conventions/.
They are used everywhere in the Material for MkDocs documentation but I'm not able to find a way to use them in my own doc. 
Thanks!
![image](https://github.com/squidfunk/mkdocs-material/assets/12848523/dcd3ad76-a21d-4a39-a0b9-07531d28d1f1)

**Comment by alexvoss** [comment, marked as answer]
This is becoming a frequently asked question, answered here, for example: https://github.com/squidfunk/mkdocs-material/discussions/6724 and here: https://github.com/squidfunk/mkdocs-material/discussions/6544#discussioncomment-7901513
